### PR TITLE
Specs: Kudos v4

### PR DESCRIPTION
**Pull request by KazanderDad**
Minor updates throughout. Improved the spam protection thinking. Added answers to most of the open questions at the bottom of the document.

### CODE REVIEW

File: specs/spec-1-kudos.md [hunk 1]
```diff
@@ -161,16 +167,24 @@ The following features will use the Social DB smart contract with the Kudos UI d
 ### Smart Contract Functions
 
 <!-- What functions and functionalities should the widget have -->
+Kudos should support two main smart contract functions:
```
Comment (KazanderDad) [inline specs/spec-1-kudos.md:170]:
```suggestion
Kudos should support two main smart contract functions:

```

**Review by robert-zaremba — COMMENTED** [review]
Added one question about the expire.

@starpause could you also review?

File: specs/spec-1-kudos.md [hunk 2]
```diff
@@ -144,14 +146,18 @@ The following features will use the Social DB smart contract with the Kudos UI d
...
+- Endorse. Sub-categories include: Developer, Community Builder, Product Lead, Marketer, Project Mgr, Sales, Researcher, Architect, Data Scientist, Creative ...
+- Nominate. Sub-categories include: House of Merit, Transparency Commissson, Counsil of Advisors, Creatives DAO, Marketing DAO
 - TODO: finalize the categories
 
 #### Spam protection
 
 - Using proof of personhood is not enough to protect a system from abuse. One user could attach a bot to his account and send thousands of kudos to other accounts (potentially to the same human account).
-- We propose to limit that activity: Any human can send maximum 14 kudos per week. The quota will reset every Sunday at midday UTC (12:00). The quota will be a parameter and we will be able to change it in the future.
-- Moreover, the target user should be in control of the Kudos they recieve. Unwanted Kudos can be removed from display by either the nominee or nominator. 
+- We propose to limit that activity: Any human can send maximum 14 kudos per week. The quota will reset every Sunday at midday UTC (12:00). The quota will be a parameter and the admin will be able to change it in the future.
```
[the remark this answers is not in the document]
Comment (KazanderDad) [reply specs/spec-1-kudos.md:158]: 14 per Week is am even 2 per day. 
It's a parameter that can be changed, the number is not important for the spec.

File: specs/spec-1-kudos.md [hunk 3]
```diff
@@ -250,49 +265,64 @@ Please include your knowledge & opinions via PR
...
 
-**Q:** Should use Social DB for upvotes?
+**Q:** Should we use Social DB for upvotes?
 
 **A:**
 
 **Q:** Should we issue SBT when someone gives a Kudo (Con: Feels spammy, costs more)? Or when someone claims their SBT after reaching a # of upvotes (Con: Less SBT will be issued)?
 
-**A:**
+**A:** @kazanderdad: Simly counting and displaying the number of Kudos received can be achieved in other ways (e.g. indexing) and automated into a separate widget that folks can show on their profile page. Minting every kudo seems unneccessary. The reason a user might want to mint SBTs would be to increase reputation in a more meaningful way, to get access or unlock new opportunities. As such, minting Kudo SBTs should be implemented together with the concept of Categories, where the admin of a Cateogory can set the required count beore minting is allowed. Example: User requires 3 kudos in the "HoM Nominee" cateogry before they can mint an official "HoM Nominee" which unlocks them to become electable in votes. But the same user might require 20 "Dev Superstar" Kudos before they can mint a "Dev Superstar" SBT, which unlocks access to some exclusive club.
```
[the remark this answers is not in the document]
Comment (KazanderDad) [reply specs/spec-1-kudos.md:280]: Good question. Maybe we will have to make the Kudos live longer, maybe a year. Or reduce the required count. In any way, these are parameters that admins can change based on community feedback.

File: specs/spec-1-kudos.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (htafolla) [reply specs/spec-1-kudos.md:158]: Can we have a whitelist to break the threshold? I have a lot of Kudos to send to NDC contributors and Champions!!

[the remark this answers is not in the document]
Comment (starpause) [reply specs/spec-1-kudos.md:158]: I prefer having no limit on the number of kudos sent which will maximize use. If we see people misbehaving then we can iterate to limit it, but MVP simplicity would be to allow any IAH verified person to send any amount.

**Pull request merged**